toolchain/sem_ir/function.h

```diff
@@ -185,50 +185,47 @@ using FunctionStore = ValueStore<FunctionId, Function>;
 
 class File;
 
-struct CalleeFunction : public Printable<CalleeFunction> {
-  // This is a C++ overload set.
-  struct CppOverloadSet {
-    // The overload set.
-    CppOverloadSetId cpp_overload_set_id;
-    // The bound `self` parameter. `None` if not a method.
-    InstId self_id;
-  };
+// The callee as a C++ overload set.
```

Comment:
I think as a comment on a top-level class, I'd go for something more context-free like
```suggestion
// Information about a callee that is a C++ overload set.
```
(and likewise for the others).

Reply:
Done